lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -1991,7 +2004,7 @@ DefConstructor('\hbox BoxSpecification HBoxContents', sub {
     if ($issvg) {    # ODDLY, svg:g isnt necessarily balanced in tikz?
       $document->maybeCloseElement('svg:g'); }
     else {
-      $document->closeNode($node); }
+      $document->closeNode($node, 1); }
```

Comment:
but yes, returning here a few months from today, I think both of us would enjoy more seeing:
```perl
       $document->maybeCloseNode($node); }
```

as a clearer signal of what we intended to develop.

Reply:
good idea; thanks!